components/automate-ui/src/app/pages/team-details/team-details.component.spec.ts

```diff
@@ -254,5 +312,14 @@ describe('TeamDetailsComponent', () => {
       });
     });
   });
-});
 
+  function genProjectOption(
+    label: string, checked?: boolean, value?: string): ProjectsFilterOption {
+    return <ProjectsFilterOption>{
+      label: label,
+      checked: checked === undefined ? false : checked,
```

Comment:
[style] same as `!!checked`, isn't it? But I guess that's more a ruby idiom than it is a JS/TS one...? 🤔